cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -234,5 +240,5 @@ private def getSkippedPaths(String[] repos) {
 }
 
 private def checkName(keepArtifacts, keepArtifactsRegex, artifactName) {
-    return !keepArtifacts || !((artifactName =~ keepArtifactsRegex).count > 0)
-}
\ No newline at end of file
+    return !keepArtifacts || (!((artifactName =~ keepArtifactsRegex).count > 0) || artifactName.contains("debug"))
```

Comment:
Why is this checking for `debug` explicitly?

Reply:
This is a quite a specific use case and actually it looks like a bypass of the configuration which is made available by regular expressions.